src/main/charts/confluence/Chart.yaml

```diff
@@ -25,6 +25,11 @@ annotations:
       links:
       - name: Github PR
         url: https://github.com/atlassian/data-center-helm-charts/pull/451
+    - kind: changed
```

Comment:
I think for this one we should set the kind as `added`?

Reply:
Thanks, fixed.